### DISCUSSION TITLE
paging-demo-for-asp-net-mvc-2/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#36] [Blog comments]
# paging-demo-for-asp-net-mvc-2/

https://blogs.taiga.nl/martijn/posts/paging-demo-for-asp-net-mvc-2/

**Comment by disqus-comments-migrator** [comment]
<em>Kim Cu commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at februari 14 2010, 11:32</em>

---

<p>Hi Martijin,</p><p>I am using your paging, It's easy to use. But I want to use it with ajax, how can I do it? Kindly to guide me.</p><p>Thanks,<br>Kim.</p>

**Comment by disqus-comments-migrator** [comment]
<em>martijn commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at februari 15 2010, 12:52</em>

---

<p><a href="#comment-397" rel="nofollow noopener" title="#comment-397">@Kim Cu </a> <br>Kim, check <a href="http://danieliglesiaseng.blogspot.com/2010/01/ajax-paging-with-aspnet-mvc.html" rel="nofollow noopener" title="http://danieliglesiaseng.blogspot.com/2010/01/ajax-paging-with-aspnet-mvc.html">http://danieliglesiaseng.blogspot.com/2010/01/ajax-paging-with-aspnet-mvc.html</a> for an AJAX version.</p>

**Comment by disqus-comments-migrator** [comment]
<em>Webdiyer commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at februari 25 2010, 07:30</em>

---

<p>Thank you for your great work! I wrote a free <a href="http://asp.net" rel="nofollow noopener" title="asp.net">asp.net</a> mvc paging component called MvcPager too,it support both standard url paging and Ajax paging using MicrosoftAjax or jQuery script library,anyone interested in this component can view online demo and download it from <a href="http://en.webdiyer.com/mvcpager" rel="nofollow noopener" title="http://en.webdiyer.com/mvcpager">http://en.webdiyer.com/mvcpager</a></p>

**Comment by disqus-comments-migrator** [comment]
<em>Kim Cu Hoang commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at februari 26 2010, 08:52</em>

---

<p>Thanks for your link, Martin.</p><p>Regards,<br>Kim.</p>
